tests/playwright/src/model/ai-lab-model-service-page.ts

```diff
@@ -66,6 +67,28 @@ export class AiModelServicePage extends AILabBasePage {
     return (await this.getAllTableRows()).length;
   }
 
+  async openServiceDetails(modelName: string): Promise<AILabServiceDetailsPage> {
+    const serviceRow = await this.getServiceByModel(modelName);
+    if (serviceRow === undefined) {
+      throw new Error(`Model [${modelName}] service doesn't exist`);
+    }
+    const serviceRowName = serviceRow.getByRole('cell').nth(3);
```

Comment:
Please open follow up issue to address missing aria-labels in the row definition. We should not rely on order of the cells. This is out of scope of this PR.